Reflection/ClassMetadataRegistry.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Pim\Bundle\CustomEntityBundle\Reflection;
+use Doctrine\Common\Util\ClassUtils;
+use Doctrine\ORM\EntityManagerInterface;
+
+
+/**
+ *
+ * @author    Romain Monceau <romain@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ */
+class ClassMetadataRegistry
```

Comment:
Need help to find better naming, better namespace

Reply:
Could you please add a description of this class. This will probably help with the naming :wink: